test/ci/buildkite/ngtf-cpu_ubuntu-external-fork.yaml

```diff
@@ -0,0 +1,121 @@
+  - command: |
+      rm -rf /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID
+      virtualenv -p /usr/bin/python3 /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv
+      source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate
+      pip install -U yapf==0.26.0
+
+    label: ":gear: Setup"
+    timeout_in_minutes: 30
+    agents:
+    - "queue=cpu-external"
+    parallelism: 1
+
+  - wait
+
+  - command: |
+      source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate
+      export PATH=/opt/llvm-3.9.0/bin/:$$PATH
+      maint/check-code-format.sh
+
+    label: ":pencil: Code Format ?"
+    timeout_in_minutes: 30
+    agents:
```

Comment:
looks like this pipeline did not exist on r0.19. @sayantan-nervana  do we need it?